wgpu-hal/src/gles/conv.rs

```diff
@@ -50,6 +50,7 @@ impl super::AdapterShared {
                 glow::RGB,
                 glow::UNSIGNED_INT_10F_11F_11F_REV,
             ),
+            Tf::R64Uint => unreachable!(),
```

Comment:
issue: Seeing a panic in this conversion makes me uneasy. Is it possible to reach this path from safe code? If so, we should probably return a validation error of some sort instead.

Reply:
theres already a panic for NV12 down below, so theres some precedent, but im not sure if this is actually unreachable

Reply:
i've followed example set by BGRA8 and just did a best effort and left a `// TODO:` instead.